src/briefcase/platforms/linux/system.py

```diff
@@ -442,7 +464,7 @@ def verify_system_packages(self, app: AppConfig):
                 f"""\
 Unable to build {app.app_name} due to missing system dependencies. Run:
 
-    sudo {system_installer} install {' '.join(missing)}
+    sudo {system_installer} {system_installer_install_flag} {' '.join(missing)}
 
 to install the missing dependencies, and re-run Briefcase.
```

Comment:
Not caused by this PR...but these values are `list`s, so the errors prints them as though they are optional.

```
Unable to build helloworld due to missing system dependencies. Run:

    sudo ['pacman'] ['-Syu'] gtk3

to install the missing dependencies, and re-run Briefcase.
```

Additionally, debian builds aren't checking for basic packages like this....so, they just bomb out later if GTK isn't installed. This may be better addressed in a separate PR, though.

Reply:
on second look....I notice this PR _is_ setting `system_installer` and `system_installer_install_flag` as lists while debian and rhel were using strings. This should be reconciled one way or the other.

Reply:
Hello
It may look like it is due to these values being lists, but it is intentional. This is because, not all users would prefer to use `pacman`. 

In systems like Manjaro, there is another package manager in addition to `pacman`, called `pamac`. `pamac` has different syntax than `pacman`. 

Ex: For installing `gtk3`, an user can do `sudo pamac install gtk3` as well as `sudo pacman -Syu gtk3`. 

So, both the `system_installer` and `system_installer_install_flag` are infact optional and can be substituted by the user as per their choice.

Reply:
That makes sense.

However, this implementation in `_system_requirement_tools` leads to two different Types of outputs: sometimes `Tuple[List[str], List[str], str, List[str]]` and sometimes `Tuple[List[str], List[str], List[str], List[str]]` (and meanwhile they're all `Optional[]`).

I'm just saying the method should really only have a single Type for its output to help prevent subtle bugs in not understanding there can be varied output Types.

Either way, though, the command that's printed for users should be runnable via copy/paste (i.e. printed without the brackets).

Reply:
You are right. It doesn't make sense that `system_installer` and `system_installer_install_flag` should be 2 different entities. So, I have updated the code  to use `system_installer_with_install_flag`. Now only one type of output will be produced: `Tuple[List[str], List[str], List[str]]`. Also, I have made the output copy/paste friendly.